src/modules/users/controller/helpers.ts

```diff
@@ -0,0 +1,20 @@
+import { ApiUserError, API_USER_ERROR_TYPE } from '../errors';
+import { UserRepository } from '../repository';
+
+export const checkForExistingLogin = async (userRepo: UserRepository, login: string) => {
```

Comment:
this logic should be re-factored using the OOP approach

Reply:
done